llvm-core/clang-runtime/metadata.xml

```diff
@@ -14,6 +14,7 @@
 		<flag name="default-compiler-rt">Use compiler-rt + libunwind instead of libgcc as the default rtlib for clang</flag>
 		<flag name="default-libcxx">Use libc++ instead of libstdc++ as the default stdlib for clang</flag>
 		<flag name="default-lld">Use lld as the default linker for clang</flag>
+		<flag name="llvm-libunwind">Use LLVM libunwind instead of GNU libunwind</flag>
```

Comment:
I don't think there's a point in adding this flag, given you're only passing it through and it wasn't present in this package before.